plugin-packs/postcss-preset-env/test/insert.after.expect.css

```diff
@@ -1,5 +1,3 @@
-$font: system-ui;
-
 .test-variable {
-	font-family: $font;
+	font-family: system-ui, -apple-system, Segoe UI, Roboto, Ubuntu, Cantarell, Noto Sans, sans-serif;
 }
```

Comment:
This change confuses me a bit. Why is this needed?

Reply:
It confused me too.

tape contents for this test :

```js
	'insert:after:array': {
		message: 'supports { stage: 1, after: { "lab-function": [ require("postcss-simple-vars") ] } } usage',
		options: {
			stage: 1,
			insertAfter: {
				'lab-function': [
					require('postcss-simple-vars')()
				]
			},
			features: {
				'lab-function': {
					unresolved: 'ignore'
				}
			}
		},
		expect: 'insert.after.expect.css'
	},
```

I replaced the `color-mod` stuff with `lab-function` to resolve the warnings.
I think that `insertAfter` looks up the index of the plugin (`lab-function`) and tries to add the other plugin after it.

If the feature is not found I think it was never inserted.

So when `color-mod` was removed `postcss-simple-vars` was no longer applied correctly in this test.

Now that I have changed it to `lab-function` it actually works again.
Although probably not as designed because of plugin (not) ordering in PostCSS 8

Reply:
Hmm we should probably add another issue to test this meaningfully since I don't think this test proves anything right now